src/main/java/uk/gov/companieshouse/overseasentitiesapi/validation/OverseasEntitySubmissionDtoValidator.java

```diff
@@ -123,6 +123,8 @@ private void validateFullRemoveDetails(OverseasEntitySubmissionDto overseasEntit
     }
 
     private void validateFullRegistrationDetails(OverseasEntitySubmissionDto overseasEntitySubmissionDto, Errors errors, String loggingContext) {
+        validateHasSecureRegister(overseasEntitySubmissionDto.getHasSecureRegister(), errors, loggingContext);
```

Comment:
Probably doesn't matter, but I put my new 'has sold land' check at the end of the method, after the existing checks.

Reply:
Whatever you like, I don't see any difference

Reply:
This is fine. Will resolve.